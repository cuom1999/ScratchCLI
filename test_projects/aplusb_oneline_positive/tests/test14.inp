66170521795112 61773002576154