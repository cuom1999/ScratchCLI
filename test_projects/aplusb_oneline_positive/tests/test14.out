127943524371266